web/satellite/src/store/index.ts

```diff
@@ -20,10 +22,12 @@ export class StoreModule<S> {
     public state: S;
     public mutations: any;
     public actions: any;
-    public getters: any;
+    public getters?: any;
```

Comment:
Why is the getters optional here?

Reply:
because some modules could not have getters